Description: Debian requires installed perl scripts to shbang perl directly
Author: Paul Gevers <elbrus@debian.org>
Forwarded: not needed

Index: lazarus/tools/delete_non_svn_files.pl
===================================================================
--- lazarus.orig/tools/delete_non_svn_files.pl
+++ lazarus/tools/delete_non_svn_files.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # *****************************************************************************
 # *                                                                           *
Index: lazarus/tools/install/replace_in_files.pl
===================================================================
--- lazarus.orig/tools/install/replace_in_files.pl
+++ lazarus/tools/install/replace_in_files.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # Copyright 2004 Mattias Gaertner
 #
